### ISSUE TITLE
Project is "deadish" - again?

### ISSUE THREAD

**Issue by Vectrex** [picturepan2/spectre#673]
To quote @picturepan2 from the discussions we had in June/July 2020:

> Here is my plan:
>
> Update basic things like devDependencies and small fixes.
> Review more Pull requests.
> Move forward to a new version.
>
> I still have to do full time work which I am in charge of product, design and dev in a startup.

I can absolutely understand, that people run out of time and have to abandon projects - but why not accept one (or more) other maintainers? Does it need another fork and the following name-calling to see some "action" again?

**Comment by salamwaddah** [comment]
shame it's not maintained. i love spectre and use it on my blog.

**Comment by eorus** [comment]
Love Spectre and it has to be maintained by the open source community. I would like to help if needed.

**Comment by jalcine** [comment]
Yeah, at this rate, I'm open to pushing up some of my changes to a forked version (worked on adding dark-mode support and leaning on CSS variables for more customization)

**Comment by qgustavor** [comment]
About working on a fork [I checked some forks](https://techgaun.github.io/active-forks/index.html#picturepan2/spectre) and seems all forks are also dead too. The fork I found that most changed Spectre was [Spectre Dark](https://kingajax.github.io/Spectre-Dark/). If forking is the solution I hope someone find a good way to manage it. I use Spectre in some of my projects, so I hope I can help implementing something.

**Comment by Vectrex** [comment]
Maybe @lu4p who did a fork in July 2020 and took it down after the dispute with @picturepan2 is willing to re-establish it.

**Comment by Vectrex** [comment]
Prodded him. @lu4p is not interested in forking any more.

**Comment by jalcine** [comment]
What dispute? Do you mind linking?

**Comment by Vectrex** [comment]
The closed issues #633 and #634 tell the story pretty much.

**Comment by inliquid** [comment]
I think it's time for a good new fork.

**Comment by cMurschall** [comment]
It's such a sad story. This framework used to be so awesome and slick. But I can totally understand that interests change and motivations fade out.
No regrets, to hard feelings. I'm grateful for the past. I don't expect any progress in this framework and don't use it in current projects anymore.

**Comment by laurentpayot** [comment]
I’m still using it in my current project because even if I had to fix myself a few bugs Spectre is still the best bang for the byte of all the many CSS frameworks I’ve tried.
Apart from the little unfixed bugs, what makes it outdated or unusable?

**Comment by zjelo** [comment]
Can some one please recommend spectree css like framework which is not dead?

Thank...

**Comment by p-baum** [comment]
@laurentpayot please fork and add your bug fixes.  You are the chosen one.

**Comment by laurentpayot** [comment]
@p-baum haha well here are all my bug fixes:

- #516
- #682

I wont fork Spectre because I’m not a CSS guy and because I replaced it with **[Beercss :beer:](https://www.beercss.com/)** for the following reasons:
- Beercss adds a modern Material Design touch that makes Spectre look outdated in comparison.
- Beercss is still new but actively maintained.  [The enhancements I proposed and the issues I raised in beercss](https://github.com/beercss/beercss/issues?q=commenter%3Alaurentpayot) were taken into account *within a couple of hours*. The maintainer is super reactive and friendly. That’s a change from Spectre!
- Beercss is almost as light as Spectre, the CSS file I get for my app is 5.4 kB (brotli)  vs 5.2 kB for Spectre (note that [I’m picking my own beercss modules](https://github.com/beercss/beercss/issues/11)).

I did not want to promote another CSS framework in this thread but considering the actually deadish state of Spectre I am just mentioning Beercss as a possible Spectre replacement.

**Comment by salamwaddah** [comment]
so anyone is up for the fork?

**Comment by KutsumaDesu** [comment]
Any good alternative?
I tried bulma and Cirrus.It's good but not handy enough.
